### PR TITLE
Adding section for extremely new users to provider development

### PR DESCRIPTION
**Pull request by crw**
Per our offline conversation, attempting to give users who want to build a function but are new to provider development a path forward, just in case they land on this page from SEO or due to direct links.

### CODE REVIEW

**Review by robin-norwood — APPROVED** [review]

File: website/docs/plugin/framework/functions/index.mdx [hunk 1]
```diff
@@ -16,6 +16,10 @@ Provider-defined function support is in technical preview and offered without co
 
 Functions are an abstraction that allow providers to expose computational logic beyond Terraform's [built-in functions](/terraform/language/functions) and simplify practitioner configurations. Provider-defined functions are supported in Terraform 1.8 and later.
 
+<Note>
+It is possible to add functions to an existing provider. If you do not have an existing provider, you will need to create your own provider to contain the functions. Please see the [Get Started](/terraform/plugin/framework/getting-started/code-walkthrough) page to learn how to create your first provider.
+</Note>
```
Comment (bendbennett) [inline website/docs/plugin/framework/functions/index.mdx:21]:
```suggestion
<Note>
It is possible to add functions to an existing provider. If you do not have an existing provider, you will need to create your own provider to contain the functions. Please see [Getting Started - Code Walkthrough](/terraform/plugin/framework/getting-started/code-walkthrough) to learn how to create your first provider.
</Note>
```

**Review by bendbennett — COMMENTED** [review]
@crw many thanks for adding this to the docs.

**Review by robin-norwood — APPROVED** [review]

**Comment by crw** [comment]
@bendbennett it looks like I need one more approval, would you mind? Thanks very much!

**Review by bendbennett — APPROVED** [review]
LGTM 🚀 

Many thanks for the contribution @crw.

**Pull request merged**